### ISSUE TITLE
How to do poker test

### ISSUE THREAD

**Issue by AhFlower** [arcetri/sts#4]
Does this project support poker test?

**Comment by lcn2** [comment]
The "poker test" is NOT directly implemented by STS, nor does it need to do so.

Several of the STS sub-test classes completely cover the ground of what is commonly referred to as the "poker test".  Moreover, instead of just dealing with a certain fixed length sequence (commonly just five symbols at a time), STS performs tests over a wide range of sequence lengths. And instead of just dealing with the limited set of standard poker hands, STS rigorously tests a complete battery of patterns including an exhaustive set of runs, aperiodic templates, etc.

In short, STS performs a far more rigorous set of tests than the common "poker test" and STS more than spans the statistical ground covered by the common "poker test".

We hope this helps.

**Comment by AhFlower** [comment]
> The "poker test" is NOT directly implemented by STS, nor does it need to do so.
> 
> Several of the STS sub-test classes completely cover the ground of what is commonly referred to as the "poker test". Moreover, instead of just dealing with a certain fixed length sequence (commonly just five symbols at a time), STS performs tests over a wide range of sequence lengths. And instead of just dealing with the limited set of standard poker hands, STS rigorously tests a complete battery of patterns including an exhaustive set of runs, aperiodic templates, etc.
> 
> In short, STS performs a far more rigorous set of tests than the common "poker test" and STS more than spans the statistical ground covered by the common "poker test".
> 
> We hope this helps.

It's works,thank you!
